test/parallel/test-dgram-send-address-types.js

```diff
@@ -5,27 +5,37 @@ const dgram = require('dgram');
 
 const buf = Buffer.from('test');
 
+const defaultCases = ['', null, undefined];
+
 const onMessage = common.mustSucceed((bytes) => {
   assert.strictEqual(bytes, buf.length);
-}, 6);
+}, defaultCases.length + 1);
 
 const client = dgram.createSocket('udp4').bind(0, () => {
   const port = client.address().port;
 
   // Check valid addresses
-  [false, '', null, 0, undefined].forEach((address) => {
+  defaultCases.forEach((address) => {
     client.send(buf, port, address, onMessage);
   });
 
   // Valid address: not provided
   client.send(buf, port, onMessage);
 
   // Check invalid addresses
-  [[], 1, true].forEach((invalidInput) => {
+  [
+    [],
+    1,
+    true,
+    false,
+    1n,
+    {},
+    Symbol(),
```

Comment:
Can you add a function to the list?
```suggestion
    Symbol(),
    () => {},
```

Reply:
We shouldn't add a function as an invalid address to test since `socket.send()`'s `address` parameter already accepts a function.
https://github.com/nodejs/node/blob/59922e3029f4bab2d27331a0919fb56f55a24d92/lib/dgram.js#L636-L639